### PR TITLE
App icon updated

### LINKED ISSUE #437: App icon for NeuroLab
**Actual Behaviour.**  
There is no app icon currently.

**Expected Behaviour**  
There should be an app icon,

**Steps to reproduce it**  
Install the app.

**Additional context**  
N/A

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by rajput2122**
Fixes #437 

**Changes**: I have changed the color of the app icon.

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[apk2.zip](https://github.com/fossasia/neurolab-android/files/3546131/apk2.zip)

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <application
         android:allowBackup="true"
         android:hardwareAccelerated="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_app_brain_icon"
```
Comment (jddeep) [inline app/src/main/AndroidManifest.xml]: Rename this to `ic_neurolab_app`

**Review by jddeep — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (rajput2122) [reply app/src/main/AndroidManifest.xml]: okk sir

**Comment by CloudyPadmal** [comment]
@rajput2122 can you make the color similar to app theme color? This one seems a bit darker.

**Comment by rajput2122** [comment]
> @rajput2122 can you make the color similar to app theme color? This one seems a bit darker.

okk sir

**Comment by jddeep** [comment]
@rajput2122 Make sure to update the PR screenshot and apk. Thanks.

**Comment by rajput2122** [comment]
> @rajput2122 Make sure to update the PR screenshot and apk. Thanks.

Sir will i have to make new pull request?

**Comment by jddeep** [comment]
> > @rajput2122 Make sure to update the PR screenshot and apk. Thanks.
> 
> Sir will i have to make new pull request?

@rajput2122 No. Just edit this PR's body from the drop down at the top right of the PR description box.

**Comment by rajput2122** [comment]
> > > @rajput2122 Make sure to update the PR screenshot and apk. Thanks.
> > 
> > 
> > Sir will i have to make new pull request?
> 
> @rajput2122 No. Just edit this PR's body from the drop down at the top right of the PR description box.

edited sir kindly review it

**Comment by rajput2122** [comment]
sir i have updated PR description box kindly review it

**Review by CloudyPadmal — APPROVED** [review]

**Comment by CloudyPadmal** [comment]
Thank you @rajput2122 for this PR. Nice work :+1:

**Review by jddeep — APPROVED** [review]
Good job @rajput2122

**Comment by rajput2122** [comment]
> Good job @rajput2122

Thank you sir

**Comment by rajput2122** [comment]
> Thank you @rajput2122 for this PR. Nice work 👍

Thank you sir

**Pull request merged**